### PR TITLE
Homework3: Renat Levotskiy

### PR DESCRIPTION
**Pull request by renat33**

### CODE REVIEW

File: zadacha1.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Напишите программу, которая печатает цифры от 1 до 100,
+# но вместо чисел, кратных 3 пишет Fizz,
+# вместо чисел кратный 5 пишет Buzz,
+# а вместо чисел одновременно кратных и 3 и 5 - FizzBuzz
+for i in range(1, 101):
+    if i % 3 == 0 and i % 5 == 0:
```
Comment (belovmd) [inline zadacha1.py]: 15?

File: zadacha2.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Используйте генератор списков: ['ab', 'ac', 'ad', 'bb', 'bc', 'bd'].
+# Используйте на предыдущий список slice: ['ab', 'ad', 'bc'].
+# Используйте генератор списков ['1a', '2a', '3a', '4a'].
+# Одной строкой удалите элемент  '2a'
```
Comment (belovmd) [inline zadacha2.py:4]: this task is missed

File: zadacha2.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Используйте генератор списков: ['ab', 'ac', 'ad', 'bb', 'bc', 'bd'].
+# Используйте на предыдущий список slice: ['ab', 'ad', 'bc'].
+# Используйте генератор списков ['1a', '2a', '3a', '4a'].
+# Одной строкой удалите элемент  '2a'
+# из прошлого списка и напечатайте его.
+# Скопируйте список и добавьте в него элемент '2a'
+# так чтобы в исходном списке этого элемента не было.
+list1 = [a + b for a in "ab" for b in "bcd"]
+list2 = list1[::2]
+list3 = [str(a) + "a" for a in range(1, 5)]
```
Comment (belovmd) [inline zadacha2.py]: '1234' ?

File: zadacha3.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# Создайте список ['a', 'b', 'c'] и сделайте из него кортеж.
+# Создайте кортеж ('a', 'b', 'c'), И сделайте из него список
+# Сделайте следующие присвоения одной строкой a = 'a', b=2, c=’python’.
+# Создайте кортеж из одного элемента, чтобы при итерировании по этому
+# элементы последовательно выводились значения 1, 2, 3.
+# Убедитесь что len() исходного кортежа возвращает 1.
+list1 = list("abc")
+tup1 = tuple(list1)
+tup2 = tuple("abc")
+list2 = tup2
```
Comment (belovmd) [inline zadacha3.py]: it's assignment. it still would be tuple but you should create list from this tuple

**Review by belovmd — COMMENTED** [review]
Please address these issues

File: zadacha2.py (hunk 2 above)
Comment (renat33) [reply zadacha2.py:4]: What task? 
print(list3.pop(1))  # пункт 4
or it could be 
print(list3.pop(list3.index("2a")))

File: zadacha2.py (hunk 3 above)
Comment (renat33) [reply zadacha2.py]: fixed

File: zadacha1.py (hunk 1 above)
Comment (renat33) [reply zadacha1.py]: fixed

File: zadacha3.py (hunk 4 above)
Comment (renat33) [reply zadacha3.py]: fixed

File: zadacha2.py (hunk 2 above)
Comment (belovmd) [reply zadacha2.py:4]: your task was firstly delete element and then add. you add it in 12 row and delete in 16th.

**Review by belovmd — COMMENTED** [review]
8,5

**Pull request closed without merging**